app_spec/test/files/links.js

```diff
@@ -1,4 +1,5 @@
 const { one, two } = require('../config')
+const sleep = require('sleep')
 
 module.exports = scenario => {
   scenario('delete_post', async (s, t) => {
```

Comment:
Why is this scenario called "delete_post"

Reply:
Woops that is not a new scenario but I have just added sleep in there :)